index.html

```diff
@@ -0,0 +1,164 @@
+      <ul class="activity_menu">
+        <li class="activity_menu_item">
+          <img src="./images/lecture.png" alt="Lecture" /><span
+            class="program_title"
+            >Introduction</span
+          >
+          <div class="brief">
+            A brief summary of the all stages for the exhibition and to take
+            full advantage of the opportunity it presents.
+          </div>
+        </li>
+        <li class="activity_menu_item">
+          <img src="./images/exhibition.png" alt="Exhibition" /><span
+            class="program_title"
+            >Exhibition</span
+          >
+          <div class="brief">
+            Presenting different categories of low cost housing to the
+            participant
+          </div>
+        </li>
+        <li class="activity_menu_item">
+          <img src="./images/forum.png" alt="Forum" /><span
+            class="program_title"
+            >Discussion</span
+          >
+          <div class="brief">
+            Open discussion between management and participant on area that need
+            more clearity
+          </div>
+        </li>
+        <li class="activity_menu_item">
+          <img
+            class="workshop"
+            src="./images/workshop.png"
+            alt="Workshop"
+          /><span class="program_title">Workshop</span>
+          <div class="brief">
+            Lecture and discussion with expert from the financial section for
+            enlightenment on different housing scheme
+          </div>
+        </li>
+        <li class="activity_menu_item">
+          <img
+            class="networking"
+            src="./images/ignite.jpg"
+            alt="Networking"
+          /><span class="program_title">Networking</span>
+          <div class="brief">
+            Opportunity to meeting new business partner for participant that
+            willing to invest.
+          </div>
+        </li>
+      </ul>
+      <p class="connect">Join Exquisite Residence Exhibition 2022</p>
+    </section>
+    <div class="guest_speaker"></div>
+    <section class="partner">
+      <h3 class="past_title">Partners</h3>
+      <div class="bar partner_bar"></div>
+      <ul class="partner_menu">
+        <li class="partner_menu_item">
+          <span><i class="fab fa-google"></i>Google</span>
+        </li>
+        <li class="partner_menu_item">
+          <span><i class="fab fa-naver"></i>NAVER</span>
+        </li>
+        <li class="partner_menu_item">
+          <span><i class="fab fa-free-apple"></i>Apple</span>
+        </li>
+        <li class="partner_menu_item">
+          <span><i class="fab fa-airbnb"></i>airbnb</span>
+        </li>
+        <li class="partner_menu_item">
+          <span><i class="fab fa-mozilla"></i>mozilla</span>
+        </li>
+      </ul>
+    </section>
+    <section class="footer">
+      <div class="footer_item company">
+        <p class="footer_name">Exquisite</p>
+        <p class="footer_name">Residence</p>

```

Comment:
- [ ] Please make sure the footer section content is in the center and increase the font size of the text content and the design matches with the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/169278176-ae27abee-a158-4320-bcef-ba33e27bd824.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/169278223-ad3bdfad-5a30-40b6-ad8b-2f6713c466a7.png)